packages/twenty-server/src/engine/core-modules/graphql/utils/generate-graphql-error-from-error.util.ts

```diff
@@ -5,13 +5,16 @@ import {
   ErrorCode,
 } from 'src/engine/core-modules/graphql/utils/graphql-errors.util';
 
-export const generateGraphQLErrorFromError = (error: Error) => {
+export const generateGraphQLErrorFromError = (
+  error: Error,
+  withStackTrace = false,
+) => {
   const graphqlError = new BaseGraphQLError(
     error.message,
     ErrorCode.INTERNAL_SERVER_ERROR,
   );
 
-  if (process.env.NODE_ENV === NodeEnvironment.development) {
+  if (process.env.NODE_ENV === NodeEnvironment.development || withStackTrace) {
```

Comment:
we can remove the NODE_ENV condition as we now always capture exceptions with stack trace